GetServiceByLatestRevision/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "services/{serviceid}/latest",
```

Comment:
why do we need this endpoint ? I think that the logic should be instead:

1) the app retrieve the list of serviceid with the latest version
2) the app calls /service/<serviceid>/<version>

the downside of having this endpoint is that the url is not 'permanent' (same url, different data)
which may cause issues with the CDN caching

Reply:
App actually calls the backend without specifying the revision version of the service receiving always the most updated version.
With this API we can keep the current app logic moving the API call from backend to the CDN without any other impact, the switch would be surely safer by the app side.

Talking about the endpoint we have already the url `/service/<serviceid>.json` used to retrieve the `service_metadata` which may cause conflicts, what would you suggest?